### ISSUE TITLE
Suggestion: pass arguments to workflow

### ISSUE THREAD

**Issue by dpo** [Khan/pull-request-comment-trigger#33]
Thank you for this action! I thought that it should be possible to use the trigger phrase to "pass arguments" to a workflow. For example, the trigger phrase "#runsomething" and the comment "#runsomething arg1 arg2" could make "arg1" and "arg2" available inside the workflow (maybe as env variables). Those could in turn be passed as arguments to scripts.

Would that be of interest? Would that be hard to implement?

**Comment by shpingalet007** [comment]
Agree, it could be nice. For example, I want to implement now a trigger to change `package.json` and `package-lock.json` version. Some sort of `/version set 0.1.2`. Maybe there would be some variations of it `/version minor +1` `/version major +1` `/version patch +1`.

**Comment by shpingalet007** [comment]
Okay, I forked this repository and made this functionality. Pushed a PR #34 if @Khan would like to accept it...

- Described `**` as a marker of argument. Can include any characters except spaces as it is considered as separator.
- Added `allow_arguments` input to the action. If not set to true, it wouldn't parse arguments, and it will expect to see `**` "as-is" if there is any in the trigger.
- Outputting to the `steps.check.outputs.arguments` as Array.

### Example
```
Trigger: /version set **
PR Comment: /version set 9.0
Output: ["9.0"]
```
